### PR TITLE
Offer command line option to clear source paths mappings

### PR DESCRIPTION
**Pull request by beckerhe**
This is a prerequisite to fix the failing E2E test. It fails because a source paths mapping already exists, while test expects an error message to pop up.

The solution: We clear all existing source paths mappings prior to starting Orbit. This could be done by directly deleting keys from the registry, but the nicer solution is in this PR: Let Orbit offer a command line option to clear this setting which means we don't have to worry in the E2E tests how and where the mappings are persisted.

### CODE REVIEW

**Review by danielfenner — APPROVED** [review]
lgtm!

File: src/OrbitQt/main.cpp [hunk 1]
```diff
@@ -66,6 +67,8 @@ ABSL_DECLARE_FLAG(bool, local);
 ABSL_DECLARE_FLAG(bool, devmode);
 ABSL_DECLARE_FLAG(bool, nodeploy);
 
+ABSL_FLAG(bool, clear_source_paths_mappings, false, "Clear all the stored sources paths mappings");
```
Comment (ronaldfw) [inline src/OrbitQt/main.cpp:70]:
```suggestion
ABSL_FLAG(bool, clear_source_paths_mappings, false, "Clear all the stored source paths mappings");
```

(Which is also what you use in other places.)

Comment (beckerhe) [reply src/OrbitQt/main.cpp:70]: Right. Thanks.

Comment (ronaldfw) [inline src/OrbitQt/main.cpp:70]: I wonder if we should have a comment that states that the main (only?) intention for usage of this flag is as part of e2e tests?

Comment (beckerhe) [reply src/OrbitQt/main.cpp:70]: A user-facing comment, or for us in the code?

Comment (ronaldfw) [reply src/OrbitQt/main.cpp:70]: For us in the code.

Comment (beckerhe) [reply src/OrbitQt/main.cpp]: I added a comment.

Comment (beckerhe) [reply src/OrbitQt/main.cpp]: I will go ahead and merge this.

**Pull request merged**